src/guide/frontend/android.md

```diff
@@ -0,0 +1,323 @@
+- [Passkeys - Google](https://developer.android.com/training/sign-in/passkeys)
+
+### Using the PasswordlessClient
+
+#### With Dagger Hilt
+
+You can either set the `ActivityContext` and `CoroutineScope` by injecting it with Dagger Hilt as follows:
+
+```kotlin
+@Module
+@InstallIn(ActivityComponent::class)
+class PasswordlessModule {
+    @Provides
+    fun provideLifecycleCoroutineScope(activity: Activity): LifecycleCoroutineScope =
+        (activity as AppCompatActivity).lifecycleScope
+
+    @Provides
+    @ActivityScoped
+    fun providePasswordlessClient(
+        @ActivityContext activity: Context, scope: LifecycleCoroutineScope): PasswordlessClient {
+        val options = PasswordlessOptions(
+            DemoPasswordlessOptions.API_KEY,
+            DemoPasswordlessOptions.RP_ID,
+            DemoPasswordlessOptions.ORIGIN,
+            DemoPasswordlessOptions.API_URL
+        )
+
+        return PasswordlessClient(options, activity, scope)
+    }
+}
+```
+
+#### Without Dagger Hilt
+
+Or you can set the Context of PasswordlessClient manually: Ensure the context is set to the current `Activity`.
+
+```kotlin
+/** Context needs to be set according to current activity
+ * If there are different activity handling register and signin,
+ * then call this on every activity
+*/
+_passwordless.setContext(this)
+```
+
+Set the coroutine scope, passing lifecycleScope of the current fragment, only necessary if you again do not use Dagger Hilt.
+
+```kotlin
+_passwordless.setCoroutineScope(lifecycleScope)
+```
+
+### Registration
+
+1. **Call Your Backend with User Details**:Make a call to your backend with user details (e.g., username, alias) and retrieve the registration token.
+2. **Call Passwordless Register Function**
+
+```kotlin
+_passwordless.register(
+    token =responseToken,
+    nickname = nickname
+) { success, exception, result ->
+        if (success) {
+            Toast.makeText(context, result.toString(), Toast.LENGTH_SHORT).show()
+        } else {
+            Toast.makeText(
+            context,
+            "Exception: " + getPasskeyFailureMessage(exception as Exception),
+        Toast.LENGTH_SHORT
+        ).show()
+    }
+}
+```
+
+### Logging in
+
+1. **Take Alias as Input**: Gather the alias as input from the user.
+2. **Call Passwordless Login**: Initiate the login process with the alias and response callback.
+
+```kotlin
+_passwordless.login(alias) { success, exception, result ->
+    if (success) {
+        lifecycleScope.launch {
+             val clientDataResponse =
+                httpClient.login(UserLoginRequest(result?.token!!))
+            if (clientDataResponse.isSuccessful) {
+                val data = clientDataResponse.body()
+                showText(data.toString())
+            }
+        }
+    } else {
+        showException(exception)
+    }
+}
+```
+
+## Sample
+
+You can find the same application [here](https://www.github.com/bitwarden/passwordless-android).
```

Comment:
Maybe have this point to the sample app? 
https://github.com/bitwarden/passwordless-android/tree/main/app

And provide a README on how to get it up and running locally